### ISSUE TITLE
Pressing 'Create new rootfs' will silently destroy your old rootfs

### ISSUE THREAD

**Issue by davidgiven** [corbinlc/gnuroot#29]
To reproduce:
1. Create a new rootfs using the default option (which for me is wheezy).
2. Launch. Modify a file. Verify the file remains when you exit and restart the rootfs.
3. Exit again.
4. Press the 'Create new rootfs' button.
5. Watch as all your precious data gets destroyed with a confirmation prompt.

Luckily I didn't actually _have_ any precious data, but it was still very scary. I was expecting a prompt for the name of the new rootfs (so letting me have multiple wheezy root file systems).

**Comment by corbinlc** [comment]
Yes.  Definitely not ideal.  This will be fixed in a future release.  Also, delete rootfs will have some safeguard added to it.
